fms2_io/readme.md

```diff
@@ -599,7 +599,19 @@ read(restart_file(3), *) date
 deallocate(restart_file)
 ```
 
-### G. FMS2_io namelist
+### H. FMS2_io namelist
 - **ncchksz:** Sets chunksize (in bytes) argument in netcdf file creation calls. The default is `64*1024`.
 - **netcdf_default_format:** Sets the netcdf file type. The acceptable values are  "64bit", "classic", "netcdf4". This can be overwritten per file if you specify `nc_format` in the open_file call. The default is 64bit.
 - **header_buffer_val:** Sets the netCDF header buffer size(in bytes). The default is 16384 bytes.
+
+### I. Compression and Chunking 
+
```

Comment:
Add something here like
In release 2022.04, "deflate_level" and the "chunksize" were added as optional arguments to the `register_diag_field`: